reference-architecture/gce-cli/gcloud.sh

```diff
@@ -367,7 +367,7 @@ if ! gcloud --project "$GCLOUD_PROJECT" compute images describe "$REGISTERED_IMA
         sleep 5
     done
     if ! gcloud -q --project "$GCLOUD_PROJECT" compute ssh "cloud-user@${TEMP_INSTANCE}" --zone "$GCLOUD_ZONE" --ssh-flag="-t" --command "sudo bash -euc '
-        subscription-manager register --username=${RH_USERNAME} --password=${RH_PASSWORD};
+        subscription-manager register --username=${RH_USERNAME} --password="\"${RH_PASSWORD}\"";
```

Comment:
@Nick-Harvey thanks for a catch and a PR, but shouldn't be `--password=\"${RH_PASSWORD}\";` enough? Could you check and possibly, update the PR? Thanks.